02_snake_animation/main.go

```diff
@@ -14,6 +14,11 @@ const (
 	snakeBgColor = termbox.ColorDefault
 )
 
+const (
+	steps = 3
+	x, y  = 10, 10 // x, y = s.pos.x(original), s.pos.y(original)
+)
+
```

Comment:
This is a very nice approach. However, I'd make this constants a part of `snake` struct. They just need better names inside `struct`, e.g. `maxSteps` instead of `steps` and `startPos` of `coord` type instead of `x` and `y`.